deploy/remote.sh

```diff
@@ -3,42 +3,43 @@
 set -e
 
 BRANCH=$1
+backend_dir=foodsaving-backend
```

Comment:
Maybe this is `project_name` and then it can be reused in more places? (ready for the next name change ;))